index.html

```diff
@@ -100,6 +100,41 @@ <h3 class="plan__price">COP 319.900/año*</h3>
         de terceros no asociados a Disney.
       </p>
     </section>
+    <!-- Devices Component -->
+    <section class="devices">
+      <h2 class="devices__title">Disponible en tus dispositivos favoritos</h2>
+      <ul class="devices-container">
+        <li class="devices-container__content">
+          <img class="devices__img" src="./img/pc.png" />
+          <h3 class="devices__options">Computadoras</h3>
+          <p class="devices__description">
+            Chrome OS<br />MacOS<br />PC Windows
+          </p>
+        </li>
```

Comment:
Let's use figure and figcaption. Please read the [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure).

Please tell me what do you think about this refactor.
```suggestion
<li class="devices-container__content">
          <figure class="device">
            <img class="device__img" src="./img/pc.png" />
            <h3 class="device__options">Computadoras</h3>
            <figcaption class="device__description">
              <ul class="devices__items">
                <li class="devices__item">Chrome OSM</li>
                <li class="devices__item">MacOS</li>
                <li class="devices__item">PC Windows</li>
              </ul>
            </figcaption>
          </figure>
        </li>
```

Reply:
I think this change will improve in the responsive design of the webpage. Please tell me if that is correct and if I am missing something about the reason of this refactor

Reply:
It’s just about semantic and accessibility